modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/JaxRsResourceRegistryImpl.java

```diff
@@ -89,6 +95,10 @@ public Object addingService(ServiceReference<Object> serviceReference) {
 
 			_jaxRsResourceProperties.put(_getClassName(object), properties);
 
+			_jaxRsModelProperties.put(
+				(String)serviceReference.getProperty("entity.class.name"),
```

Comment:
According to my approach, here we should fill the `_jaxRsModelProperties` with all the properties.